flake.nix

```diff
@@ -1,5 +1,5 @@
 {
-  description = "srid/haskell-template: Nix template for Haskell projects";
+  description = "srid/mood-tracker: Nix template for Haskell projects";
```

Comment:
Is "Nix template for Haskell projects" still a valid description for this flake?